From a2c99456f92df14efeabd8eee37a9af2eb115f18 Mon Sep 17 00:00:00 2001
From: robototechnic <anon@post.su>
Date: Sun, 11 Feb 2018 11:12:51 +0300
Subject: [PATCH] tools enable building of zyimage

---
 tools/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tools/Makefile b/tools/Makefile
index 50bd552d75..508c593215 100644
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -37,6 +37,7 @@ tools-$(BUILD_B43_TOOLS) += b43-tools
 tools-$(BUILD_ISL) += isl
 tools-$(CONFIG_USE_SPARSE) += sparse
 tools-$(CONFIG_TARGET_apm821xx) += genext2fs
+tools-$(CONFIG_TARGET_ramips) += zyimage
 
 # builddir dependencies
 $(curdir)/bison/compile := $(curdir)/flex/compile
@@ -71,6 +72,7 @@ $(curdir)/libressl/compile := $(curdir)/pkg-config/compile
 $(curdir)/mkimage/compile += $(curdir)/libressl/compile
 $(curdir)/firmware-utils/compile += $(curdir)/libressl/compile
 $(curdir)/cmake/compile += $(curdir)/libressl/compile
+$(curdir)/zyimage/compile := $(curdir)/findutils/install
 
 ifneq ($(HOST_OS),Linux)
   tools-y += coreutils
-- 
2.11.0

