From c8339ec3b86df58335ddd2379322eb4efa88315e Mon Sep 17 00:00:00 2001
From: robototechnic <anon@post.su>
Date: Sun, 11 Feb 2018 11:43:32 +0300
Subject: [PATCH] ramips add some pins to mt7620n

---
 target/linux/ramips/dts/mt7620n.dtsi | 46 ++++++++++++++++++++++++++++++++++++
 1 file changed, 46 insertions(+)

diff --git a/target/linux/ramips/dts/mt7620n.dtsi b/target/linux/ramips/dts/mt7620n.dtsi
index 5d3c1e3fa4..062e825ef4 100644
--- a/target/linux/ramips/dts/mt7620n.dtsi
+++ b/target/linux/ramips/dts/mt7620n.dtsi
@@ -228,6 +228,13 @@
 		state_default: pinctrl0 {
 		};
 
+		i2c_pins: i2c {
+			i2c {
+				ralink,group = "i2c";
+				ralink,function = "i2c";
+			};
+		};
+
 		ephy_pins: ephy {
 			ephy {
 				ralink,group = "ephy";
@@ -249,12 +256,51 @@
 			};
 		};
 
+		uartf_pins: uartf {
+			uartf {
+				ralink,group = "uartf";
+				ralink,function = "uartf";
+			};
+		};
+
 		uartlite_pins: uartlite {
 			uart {
 				ralink,group = "uartlite";
 				ralink,function = "uartlite";
 			};
 		};
+
+		wdt_pins: wdt {
+			wdt {
+				ralink,group = "wdt";
+				ralink,function = "wdt";
+			};
+		};
+		pa_pins: pa {
+			pa {
+				ralink,group = "pa";
+				ralink,function = "pa";
+			};
+		};
+		mdio_pins: mdio {
+			mdio {
+				ralink,group = "mdio";
+				ralink,function = "mdio";
+			};
+		};
+		rgmii1_pins: rgmii1 {
+			rgmii1 {
+				ralink,group = "rgmii1";
+				ralink,function = "rgmii1";
+			};
+		};
+		phy_led_pins: phy_led {
+			phy_led {
+				ralink,group = "led";
+				ralink,function = "led";
+			};
+		};
+
 	};
 
 	rstctrl: rstctrl {
-- 
2.11.0

